.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,16 @@
+name: "Commit Lint"
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v1
+        with:
+          failOnWarnings: true.githu
```

Comment:
The value is a bit weird. Is it ok ?

Reply:
I agree, it should be `true`. Seems like an accidental copy/paste.